src/_data/catalog/widgets.json

```diff
@@ -1608,5 +1608,17 @@
     ],
     "link": "https://docs.flutter.io/flutter/widgets/AnimatedWidgetBaseState-class.html",
     "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "DropdownButton",
+    "description": "A input field to choose an item from a list.",
```

Comment:
"A input field to choose..." => "An input field for choosing..."

I thought the idea was to add Spinner to this description for Android users?